src/index.html

```diff
@@ -9,6 +9,6 @@
   <body>
     <noscript> You need to enable JavaScript to run this app. </noscript>
     <div id="root"></div>
-    <script src="./index.js"></script>
+    <script src="./index.tsx"></script>
```

Comment:
This alone sold me on Parcel. Took zero configuration to replace with TS.

Reply:
I've been happy with Parcel over CRA.  Been doing much more Next.js and don't have a a good idea of what the latest news is for either of them anymore.

Reply:
That's how I'm deploying as well - actually Next on Netlify. I want the whole package - GraphQL, TS, Netlify, Next.

Reply:
I viewed a React class from Brian Holt in late 2017.  He set everything up from scratch and bundled with Parcel.